src/engine/enginebuffer.cpp

```diff
@@ -1387,7 +1387,17 @@ void EngineBuffer::updateIndicators(double speed, int iBufferSize) {
     m_iSamplesSinceLastIndicatorUpdate += iBufferSize;
 
     const double fFractionalPlaypos = fractionalPlayposFromAbsolute(m_playPosition);
```

Comment:
if you rename m_slipPosition to m_slipPos , also rename m_playPosition to m_playPos